doc/lsst.meas.base/tasks/lsst.meas.base.forcedMeasurement.ForcedMeasurementTask.rst

```diff
@@ -0,0 +1,36 @@
+.. lsst-task-topic:: lsst.meas.base.forcedMeasurement.ForcedMeasurementTask
+
+#####################
+ForcedMeasurementTask
+#####################
+
+``ForcedMeasurementTask`` runs a set of (user-selectable) plugins, implementing particular measurement algorithms, to measure the properties of sources on a single image.
+A reference catalog, containing measurements made elsewere, is used to constrain some aspects of the algorithms.
+
+Processing Summary
+==================
+
+The set of plugins to run is set in the task configuration.
+Each plugin defines the the values that it measures (which correspond to columns in the output table), and then conducts measurement on each detected source.
+See `~lsst.meas.base.ForcedPlugin` for details.
```

Comment:
Do you really need `~lsst.meas.base.` prefix? I find if I omit the prefix it still works, due to the `.. py:currentmodule:: lsst.meas.base` directive which I assume you are using.